README.md

```diff
@@ -13,33 +13,41 @@ Your first PR should be a first draft of the final CAIP. An editor will manually
 * **CAIP-19** - [Asset Type and Asset ID Specification](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md)
 * **CAIP-20** - [Asset Reference for the SLIP44 Asset Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-20.md)
 * **CAIP-21** - [Asset Reference for the ERC20 Asset Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-21.md)
 * **CAIP-22** - [Asset Reference for the ERC721 Asset Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-22.md)
-* **CAIP-23** - [Blockchain Reference for Filecoin Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-23.md)
 * **CAIP-25** - [Chain Agnostic Provider Handshake](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-25.md)
-* **CAIP-26** - [Blockchain Reference for the Tezos Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-26.md)
 * **CAIP-27** - [Chain Agnostic Provider Request](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-27.md)
-* **CAIP-28** - [Blockchain Reference for the Stellar Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-28.md)
 * **CAIP-29** - [Asset Reference for the ERC1155 Asset Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-29.md)
-* **CAIP-30** - [Blockchain Reference for the Solana Namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-30.md)
 * **CAIP-74** - [CACAO: Chain Agnostic CApability Object](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-74.md)
-* **CAIP-76** - [Account Address for the Hedera namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-76.md)
+* **CAIP-76** - [Account Address for the Hedera namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-76.md) - see [chainagnostic/namespaces#16](https://github.com/ChainAgnostic/namespaces/pull/16)
+* **CAIP-104** - [Namespace Reference Purpose and Guidelines](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-104.md)
+
+## Namespaces
+
+Please note that CAIPs specific to a blockchain community or namespace have been moved to a separate repository, <a href=https://github.com/chainagnostic/namespaces>`namespaces`</a>, governed by the same processes.  CAIPs 
+[3](https://github.com/ChainAgnostic/namespaces/tree/main/eip155), 
+[4](https://github.com/ChainAgnostic/namespaces/tree/main/bip122), 
+[5](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-5.md)),
+[23](https://github.com/ChainAgnostic/namespaces/tree/main/eip155), 
+[26](https://github.com/ChainAgnostic/namespaces/tree/main/tezos), 
+[28](https://github.com/ChainAgnostic/namespaces/tree/main/stellar), and 
+[30](https://github.com/ChainAgnostic/namespaces/tree/main/solana) 
+have been migrated. CAIPs
+[6](https://github.com/ChainAgnostic/namespaces/pull/6),
+[7])(https://github.com/ChainAgnostic/namespaces/pull/5), and
+[13]()
+are still in process of migration.
```

Comment:
Is this enough to close #116 , @oed , or should it be more detailed?

Reply:
I still think it 's good practice to at least mention the CAIPs that have disappeared, in case someone comes looking for them; I take your point about pointing to open PRs, tho! Whatever version of this people prefer more, I'd support merging on today's call to wrap this all up.

```suggestion

## Namespaces

Previously, there were a number of CAIPs that have been migrated to the *namespaces* repository: 3 (eip155), 4 (bip122), 5 (cosmos), 6 (polkadot), 7 (eosio), 13 (), 23, 26, 28 and 30. Chain Agnostic [Namespaces](https://github.com/chainagnostic/namespaces) describe a blockchain ecosystem or set of ecosystems as a namespace, relying as much as possible on the CAIP specifications to minimize the research needed to interact with assets, contracts, and accounts in that namespace. 
```

Reply:
I just find it kind of weird to treat the earlier namespaces differently from new ones, e.g. giving them special treatment here.

Reply:
only to tell people who come looking for something they read a year ago and find half the list missing!

Reply:
i'm ok with either version, tho

Reply:
I think it's clear enough to say that some CAIPs have been migrated! if they don't see the one they are looking for they will likely click the link anyway!